api/src/pcapi/scripts/remove_product_ean_duplicate/main.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+from itertools import groupby
+
+from sqlalchemy.orm import joinedload
+
+from pcapi import settings
+from pcapi.core.offers import models as offers_models
+from pcapi.models import db
+
+
+def remove_product_ean_duplicate(do_update: bool) -> None:
+    products = (
+        offers_models.Product.query.options(joinedload(offers_models.Product.offers))
+        .filter(~offers_models.Product.extraData["ean"].astext.is_(None))
```

Comment:
Pour info, tu peux plutôt faire
```suggestion
        .filter(offers_models.Product.extraData["ean"].astext.is_not(None))
```